src/css/admin/form-builder.css

```
@@ -0,0 +1,59 @@
+    @apply wpuf-transition-all wpuf-duration-100 wpuf-ease-out wpuf-opacity-0 wpuf-scale-75;
+}
+
+.wpuf-dropdown-container:hover .wpuf-dropdown-item {
+    @apply wpuf-opacity-100 wpuf-scale-100;
+}
+
+.wpuf-pattern-1 {
+    border: 1px dashed #ddd;
+    background-color: #ffffff;
+    opacity: 1;
+    background: repeating-linear-gradient( -45deg, #dcfce7, #dcfce7 2px, #ffffff 2px, #ffffff 6px );
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)